paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/ActionBase.java

```diff
@@ -79,11 +79,9 @@ public abstract class ActionBase implements Action {
     }
 
     ActionBase(String warehouse, String databaseName, String tableName, Options options) {
+        options.set(CatalogOptions.WAREHOUSE, warehouse);
```

Comment:
We need specify the options is for catalog, and table options can be handled in action implementation.